compose/src/main/res/values/strings.xml

```diff
@@ -59,6 +59,30 @@
     </plurals>
     <string name="today">Today</string>
 
+    <!-- Snippet Details -->
+    <string name="created_by">Created By %1s</string>
+    <string name="owned_by">Owned By</string>
+    <string name="last_modified">Last Modified</string>
+    <string name="header_snippet_files">Snippet Files</string>
+    <string name="header_snippet_comments">Snippet Comments %1s</string>
```

Comment:
@MMurrayAtBR Append string name with `_format` suffix to make it clear to the consumer of this string that it should require format args :)